src/app/features/tech-record/components/change-vehicle-type/change-vehicle-type.component.html

```diff
@@ -0,0 +1,59 @@
+<div *ngIf="vehicleTechRecord">
+  <br />
```

Comment:
css on the div or h2 would be preferable to this as `<br/>` is not semantically correct in this case.

Reply:
removed